mlflow/sklearn/__init__.py

```diff
@@ -668,7 +671,25 @@ def fit_mlflow(self, func_name, *args, **kwargs):
             else:
                 try_mlflow_log(mlflow.log_metric, "training_score", training_score)
 
-        try_mlflow_log(log_model, self, artifact_path="model")
+        SAMPLE_ROWS = 5
+        fit_arg_names = _get_arg_names(self.fit)
+        X_var_name, y_var_name = fit_arg_names[:2]
+        X_sample = _get_Xy(args, kwargs, X_var_name, y_var_name)[0][:SAMPLE_ROWS]
```

Comment:
How many rows should we sample here?

Reply:
I think that 5 rows makes sense to me! Can we try/catch this block on the off chance that slice-based sampling of the data fails? I know that sklearn indicates that data should be array-like, but it'd be nice to cover the unlikely case where it isn't.